src/contexts/GameContext.js

```diff
@@ -3,8 +3,8 @@ import React, { createContext, useState, useEffect } from "react";
 import { io } from "socket.io-client";
 import { uniqueNamesGenerator, colors, animals } from "unique-names-generator";
 
-// const socket = io("localhost:4000");
-const socket = io("https://luzhanqi.herokuapp.com/");
+const socket = io("localhost:4000");
```

Comment:
The backend url could be abstracted into a config file which in turn reads from an environmental variable.